cypress/e2e/buy-shirt.cy.ts

```diff
@@ -26,13 +27,15 @@ describe("Buy a t-shirt", () => {
     shippingStepPage = new ShippingStepPage();
     paymentStepPage = new PaymentStepPage();
 
+    tShirtName = "Faded Short Sleeve T-shirts";
```

Comment:
use a variable name to indicate the product you are referring to, something like: sleeveTShirt or faddedShortSleeve, etc.